kobuki_node/src/odometry.cpp

```diff
@@ -55,8 +66,14 @@ bool Odometry::commandTimeout(const rclcpp::Time & now) const {
   }
 }
 
+void Odometry::resetTimeout(const rclcpp::Time & now)
+{
+  last_cmd_time_ = now;
```

Comment:
General migration of inline code to source code? We used to be in the habit of optimising on arm cores any one-liner to an inline call. Not really an issue for kobuki though, just wondering if there was a significant reason you do so here?

Reply:
Just consistencies sake; for me, it is easier to just read the C++ file for implementation and not have to open the header file unless I'm changing signatures.